packages/plugins/graphql-executor/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@envelop/graphql-executor",
+  "version": "0.0.0",
+  "description": "A customizable GraphQL Spec compliant Executor",
+  "license": "MIT",
+  "author": "Saihajpreet Singh <saihajpreet.singh@gmail.com> (https://saihaj.dev/)",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dotansimha/envelop.git",
+    "directory": "packages/plugins/graphql-executor"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs"
+    },
+    "./*": {
+      "require": "./dist/*.js",
+      "import": "./dist/*.mjs"
+    }
+  },
+  "typings": "dist/index.d.ts",
+  "typescript": {
+    "definition": "dist/index.d.ts"
+  },
+  "scripts": {
+    "test": "jest",
+    "prepack": "bob prepack"
+  },
+  "dependencies": {
+    "graphql-executor": "^0.0.8"
```

Comment:
Should this be a peer dependency?

Reply:
I can, but I'm not 100% sure.
I think we can depend on a specific version because this is something users might not install for a different use case?

Reply:
You can have multiple versions of graphql executor in a project, so no problem being a regular dependency from that angle (as opposed to graphql js) but maybe you are asking based on other concerns in terms of envelop plug-ins

Reply:
I think having it as a dep is probably easier for now until we get to a stable release similar to how we have `graphql-jit`